_data/projects/fibonacci-series.yml

```
@@ -0,0 +1,13 @@
+name: fibonacci-series
+desc: This repository is to write program the famous Fibonacci series in as many different programming languages as we can.
+site: https://github.com/fharookshaik/fibonacci_series
```

@fharookshaik I believe you changed the Repo name to https://github.com/fharookshaik/fibonacci-series. please correct.

```suggestion
site: https://github.com/fharookshaik/fibonacci-series
```